src/webapp/github.py

```diff
@@ -49,7 +109,7 @@ def github_api_call(self, method, url, data):
         self._add_auth_header(req)
         #add_gh_preview_header(req)
         try:
-            resp = urllib.request.urlopen(req)
+            resp = urllib.request.urlopen(req, cafile=certifi.where())  # potentially need cafile=certifi.where()
```

Comment:
```suggestion
            # potentially need cafile=certifi.where()
            resp = urllib.request.urlopen(req, cafile=certifi.where())
```

:thinking: just curious did you run into a case where you needed this?

Also @CannonLock, it's about time I tell you because nobody else on the team will.

The 79-wide rule is the most important rule in PEP-8.